suites/pacific/cephadm/test_5-2_new-feature.yaml

```diff
@@ -0,0 +1,138 @@
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: Cephadm Bootstrap with custom dashboard credentials
+      desc: Bootstrap with initial-dashboard user and password option
+      module: test_bootstrap.py
+      config:
+        command: bootstrap
+        base_cmd_args:
+          verbose: true
+        args:
+          registry-url: registry.redhat.io
+          custom_image: true
+          mon-ip: "node1"
+          orphan-initial-daemons: true
+          skip-monitoring-stack: true
+          initial-dashboard-user: admin123
+          initial-dashboard-password: admin123
+          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Add host
+      desc: Add new host node with IP address and labels
+      module: test_host.py
+      config:
+        service: host
+        command: add
+        args:                     # arguments to ceph orch
+          attach_ip_address: true
+          node: node2
+          labels:
+            - mon
+            - mgr
+            - osd
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Add host
+      desc: Add new host node with IP address and labels
+      module: test_host.py
+      config:
+        service: host
+        command: add
+        base_cmd_args:
+          verbose: true
+        args:
+          node: node3
+          attach_ip_address: true
+          labels:
+            - mon
+            - osd
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Apply Monitor with host placement
+      desc: Apply monitor with placement option on specified hosts
+      module: test_mon.py
+      abort-on-fail: true
+      config:
+        command: apply
+        service: mon
+        base_cmd_args:
+          verbose: true
+        args:
+          placement:
+            nodes:
+                - node1
+                - node2
+                - node3
+            sep: ";"
+  - test:
+      name: Apply Manager service
+      desc: Apply manager service with placement option
+      module: test_mgr.py
+      config:
+        command: apply
+        service: mgr
+        base_cmd_args:
+          verbose: true
+        args:
+          placement:
+            nodes:
+              - node1
+              - node2
+            sep: " "
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Cephadm support for raw mode OSDs using CLI
+      desc: Add raw osd on node3
+      module: test_daemon.py
+      polarion-id: CEPH-83574990
+      config:
+        command: add
+        service: osd
+        pos_args:
+          - "node3"
+          - "/dev/vdb"
+        args:
+          method: raw
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
```

Comment:
Can we add few basic IO tests which might run for a minute or so after deploying raw mode OSDs? 

@sunilkumarn417  Do you think that might add some value add to the tests?

Reply:
Yes. but some more OSDS would be good to test IOs.